Source/CsDebugScript.Engine/Process.cs

```diff
@@ -543,6 +544,25 @@ public int FindMemoryRegion(ulong address)
         }
 
         /// <summary>
+        /// Invalidates cache structures.
+        /// Use when memory state changes (e.g. during live debugging).
+        /// </summary>
+        public void InvalidateProcessCache()
```

Comment:
Please take a look at attempt after iteration 24. I am open for suggestions on how it can be improved.

---
In reply to: [69979083](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r69979083) [](ancestors = 69979083)